lib/core/src/sdk.rs

```diff
@@ -52,6 +52,14 @@ pub const DEFAULT_DATA_DIR: &str = ".data";
 /// Number of blocks to monitor a swap after its timeout block height
 pub const CHAIN_SWAP_MONITORING_PERIOD_BITCOIN_BLOCKS: u32 = 4320;
 
+/// A list of external input parsers that are used by default.
+/// To opt-out, set `use_default_external_input_parsers` in [Config] to false.
+pub const DEFAULT_EXTERNAL_INPUT_PARSERS: &[(&str, &str, &str)] = &[(
```

Comment:
Do we need it to be public? I see it is only used inside the sdk.rs

Reply:
I've only made it public because it's cool to see and link to the constant in the docs.